src/operator/tensor/indexing_op.cc

```diff
@@ -129,7 +186,7 @@ NNVM_REGISTER_OP(one_hot)
 .set_attr<nnvm::FInferType>("FInferType", OneHotOpType)
 .set_attr<FCompute>("FCompute<cpu>", OneHotOpForward<cpu>)
 .set_attr<nnvm::FGradient>("FGradient", MakeZeroGradNodes)
-.add_argument("indices", "NDArray", "array of locations where to set on_value")
+.add_argument("indices", "ndarray-or-symbol", "array of locations where to set on_value")
```

Comment:
you mean onehot needs ndarray indices? or this type info is required by frontend for most ops?